### PR TITLE
Fix: Don't report issues with no instances (after aderyn ignore was introduced)

### PR DESCRIPTION
**Pull request by TilakMaddy**
After running through ignore lines, if there are 0 issues, then don't include it in report

### CODE REVIEW

File: aderyn_core/src/lib.rs [hunk 1]
```diff
@@ -168,6 +168,11 @@ where
                         .is_some_and(|lines_to_ignore| lines_to_ignore.contains(&instance.1))
                 })
                 .collect();
+
+            if issue.instances.is_empty() {
```
Comment (alexroan) [inline aderyn_core/src/lib.rs:172]: Test for this?

**Review by alexroan — CHANGES_REQUESTED** [review]

**Comment by TilakMaddy** [comment]
Waiting for #704  to get in, so I can write higher quality tests

File: reports/empty_report.md
Comment (TilakMaddy) [inline reports/empty_report.md:1]: Here !

File: aderyn_core/src/lib.rs (hunk 1 above)
Comment (TilakMaddy) [reply aderyn_core/src/lib.rs:196]: ✅ DONE

**Review by alexroan — APPROVED** [review]

**Pull request merged**